tests/steps/steps.py

```diff
@@ -6,19 +6,39 @@
 
 @given("Account is created and added to the database")
 def step_impl(context):
-    pass  # Placeholder for account creation
+    url = BASE_URL + "accounts/"
+    data ={
+        "initial_balance": 2000,
+        "owner": "Greg"
+    }
+    response = requests.post(url, json=data)
```

Comment:
The code could be refactored to make it more reusable and maintainable, especially if you then be adding more steps or scenarios. You can create helper function to reduce redundancy.

def create_account(initial_balance, owner):
    url = BASE_URL + "accounts/"
    data = {
        "initial_balance": initial_balance,
        "owner": owner
    }
    response = requests.post(url, json=data)
    return response
    
    and then call this function inside of the step:
    
   @given("Account is created and added to the database")
def step_impl(context):
    response = create_account(initial_balance=2000, owner="Greg")
    json_data = response.json()
    context.status_code = response.status_code
    context.id = json_data["id"]
    context._expected_balance = json_data["balance"]
    
    
    Note: helper functions should be placed inside of the helpers folder. Not inside steps.py
    So create the folder named "helpers" under the "tests". Inside of the helpers folder create 2 python files:
    __init__.py and helpers.py and place helper functions in helpers.py. Then import it to the steps.py